app/iht/views/deadlines.scala.html

```diff
@@ -0,0 +1,29 @@
+@*
+* Copyright 2016 HM Revenue & Customs
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*   http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*@
+
+@import uk.gov.hmrc.play.views.html._
+@import iht.views.html._
+@import iht.controllers.ControllerHelper
+@import iht.utils._
+@import iht.constants.IhtProperties
+@import uk.gov.hmrc.play.partials.FormPartialRetriever
+
+@(implicit request:Request[_], messages: Messages, ihtFormPartialRetriever: FormPartialRetriever)
+    <p>@messages("page.iht.deadlines.p1")</p>
+    <p>@messages("page.iht.deadlines.p2") <a id="pay-early" href="@IhtProperties.linkPayEarly">@messages("page.iht.deadlines.anchorText")</a>.</p>
+    <p>@messages("page.iht.deadlines.p3")</p>
+
+    <a id="exit-to-iht400" class="button" href="@IhtProperties.linkEstateReportKickOut">@messages("page.iht.filter.paperform.million.exit")</a>
```

Comment:
This should probably exit to the [Welsh IHT400 page](https://www.gov.uk/government/publications/inheritance-tax-inheritance-tax-account-iht400.cy) if the service is currently in Welsh

Reply:
Fixed